webapp/src/components/createCategory/createCategory.tsx

```diff
@@ -1,9 +1,9 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React, {useState} from 'react'
+import React, { useState } from 'react'
```

Comment:
Were these changes caused by the linter? If not, we should revert them.

Reply:
This was because of my mistake, I have used vscode built-in linter, I will revert this back. Thank you.